plugins/polio/js/src/domains/Calendar/campaignCalendar/map/CampaignsLegend.tsx

```diff
@@ -1,64 +1,99 @@
-import React, { FunctionComponent } from 'react';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import {
-    Grid,
-    Typography,
     Accordion,
-    AccordionSummary,
     AccordionDetails,
+    AccordionSummary,
     Box,
+    Grid,
+    Typography,
 } from '@mui/material';
 import { useSafeIntl } from 'bluesquare-components';
-import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
-import { useStyles } from '../Styles';
+import React, { FunctionComponent } from 'react';
+import { vaccineOpacity } from '../Styles';
 
+import { SxStyles } from '../../../../../../../../hat/assets/js/apps/Iaso/types/general';
 import MESSAGES from '../../../../constants/messages';
 import { MappedCampaign } from '../types';
 
 type Props = {
     campaigns: MappedCampaign[];
 };
 
+const styles: SxStyles = {
+    title: {
+        height: theme => theme.spacing(6),
+        minHeight: theme => theme.spacing(6),
+        '&.Mui-expanded': {
+            minHeight: theme => theme.spacing(6),
+        },
+    },
+    details: {
+        paddingTop: 0,
+    },
+    mapLegendCampaigns: {
+        width: 165,
+    },
```

Comment:
This is just copied from `useStyles`. I don't see what we gain from moving  this styling to `sx` if nothing changes. Now if we need to change this styling we have to do it in 2 places

Reply:
i prefere move to sx has we don't have to use important anymore, sx is adding directly style to mui styilng, avoiding css hierarchy problem. Unfortunately mapLegendCampaigns us used somewhere alse, so i kept it